.gitignore

```diff
@@ -1,11 +1,15 @@
 .idea
+.firebase
+.runtimeconfig.json
+
 */node_modules
 */coverage-final.json
-*/package-lock.json
-*/firebase-debug.log
-*/firestore-debug.log
+
+package-lock.json
```

Comment:
There's no top-level package.json, hence there should be no package-lock.json... is this ignore required?